ext/soap/tests/typemap014.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+SOAP typemap 14: typemap bad from_xml callback raise segfault
```

Comment:
Should probably use the bug title, and the file should be placed in the `bugs` directory.

Reply:
done